cla_backend/settings/docker.py

```diff
@@ -20,30 +21,27 @@
 
 MANAGERS = ADMINS
 
-#HOST_NAME = "http://"
-
-# LOGGING CONFIG FOR DOCKER ENV
 LOGGING['handlers']['production_file'] = {
-    'level' : 'INFO',
-    'class' : 'logging.handlers.RotatingFileHandler',
-    'filename' : '/var/log/wsgi/app.log',
-    'maxBytes': 1024*1024*5, # 5 MB
-    'backupCount' : 7,
+    'level': 'INFO',
+    'class': 'logging.handlers.RotatingFileHandler',
+    'filename': '/var/log/wsgi/app.log',
+    'maxBytes': 1024 * 1024 * 5,  # 5 MB
+    'backupCount': 7,
     'formatter': 'logstash',
     'filters': ['require_debug_false'],
 }
 
 LOGGING['handlers']['debug_file'] = {
-    'level' : 'DEBUG',
-    'class' : 'logging.handlers.RotatingFileHandler',
-    'filename' : '/var/log/wsgi/debug.log',
-    'maxBytes': 1024*1024*5, # 5 MB
-    'backupCount' : 7,
+    'level': 'DEBUG',
+    'class': 'logging.handlers.RotatingFileHandler',
+    'filename': '/var/log/wsgi/debug.log',
+    'maxBytes': 1024 * 1024 * 5,  # 5 MB
+    'backupCount': 7,
     'formatter': 'verbose',
     'filters': ['require_debug_true'],
 }
 
 LOGGING['loggers'][''] = {
-    'handlers': ['production_file', 'debug_file'],
+    'handlers': ['production_file', 'debug_file', 'console'],
```

Comment:
🙋‍♂️ Can we remove both `production_file` and `debug_file` handlers in favour of `console`? If we need to add more granular logging, we could add `LOG_LEVEL` environment variable to switch around.

Reply:
Sure, happy to. Was minimising changes, but would love to start delicately spring cleaning soon.

Reply:
I'm happy either way 👍